src/app/public/src/modules/layout/layout-sidebar.component.html

```diff
@@ -8,7 +8,7 @@
     </stache-edit-button>
 
     <stache-row reverseColumnOrder="true">
-      <stache-column screenSmall="8" screenMedium="9" screenLarge="10">
+      <stache-column screenSmall="12" screenMedium="12" screenLarge="12">
```

Comment:
There's no reason to use `stache-row` and `stache-column` if you're using `12` for all of the breakpoints.

Reply:
I can remove this now, but I had left it in here because these columns will be adjusted once we move the TOC/in-page nav